### ISSUE TITLE
Missing documentation for "Available Response methods?"

### ISSUE THREAD

**Issue by mdemoss** [spantaleev/sijax-python#11]
In the section for [Available Response methods](https://pythonhosted.org/Sijax/usage.html#available-response-methods) there a link pointing to [BaseResponse](https://pythonhosted.org/Sijax/api.html#available-response-methods) saying the list of available response methods can be found there, but that section is empty.

It looks like a section documenting the response methods was removed in [this change](https://github.com/spantaleev/sijax-python/commit/83c6f15aa60278ebeeecb5d2ca033dc511058529#diff-2c850fc4dacde89c4c471360aeadeaddL52).

**Comment by eccentric** [comment]
Yeah, that seems important. I just spent some time looking for that documentation only to finally arrive here. Looks like they were outlined pretty well before this update to move to sphinx.
